src/city_test.cc

```diff
@@ -20,14 +20,15 @@
 #include "mjolnir/util.h"
 #include "odin/directionsbuilder.h"
 #include "odin/util.h"
-#include "proto/directions_options.pb.h"
-#include "proto/tripdirections.pb.h"
-#include "proto/trippath.pb.h"
 #include "sif/costfactory.h"
 #include "thor/bidirectional_astar.h"
 #include "thor/pathalgorithm.h"
 #include "thor/trippathbuilder.h"
 
+#include <valhalla/proto/directions_options.pb.h>
```

Comment:
The system-style include here is because they are auto-generated?

Reply:
yes, it is good  to have generated sources at some system-like location, but it is not a strict rule